### PR TITLE
Update SearchPage.js

### PR DESCRIPTION
**Pull request by EAguayodev**
The following code is what you suggested be added in the return function

### CODE REVIEW

File: Cohort1/tbd/client/tbd/src/search/SearchPage.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
+import './index.css';
```
Comment (allaboutmike) [inline Cohort1/tbd/client/tbd/src/search/SearchPage.js]: These files are missing, which is why the netlify build is complaining. You probably don't need both the index.css and search.css file. So maybe combine them into one file and add it here.

File: Cohort1/tbd/client/tbd/src/search/SearchPage.js [hunk 2]
```diff
@@ -1,7 +1,14 @@
+import './index.css'; 
+import '.search.css';
```
Comment (allaboutmike) [inline Cohort1/tbd/client/tbd/src/search/SearchPage.js:1]: This should probably not have the leading `.`; so it would be `import 'search.css'` instead.

**Review by allaboutmike — CHANGES_REQUESTED** [review]

**Comment by EAguayodev** [comment]
Fixed changes suggested. Waiting for the code to get approved, included search.css and took off App.css import

**Comment by allaboutmike** [comment]
The search.css wasn't included in the commit. May need to add it or make sure it's tracked in git. Once we add that in this should work.

**Pull request closed without merging**